### ISSUE TITLE
SyntaxError: Unexpected token 'B', "Billing ha"... is not valid JSON

### ISSUE THREAD

**Issue by prasadshembekar5** [adrianhajdin/project_ai_mern_image_generation#28]
this error comes again and again, although i have used new api key from my personal account 

![image](https://user-images.githubusercontent.com/89524884/218050275-ebd1b2a4-4e5d-46fd-bc4d-0eb86a0aa935.png)

**Comment by alsyferr** [comment]
I have same problem , have you found a solution?

**Comment by prasad-shembekar** [comment]
Let's wait, if someone gets solution of this issue

**Comment by Ugyenwangdi** [comment]
We need to sign up for new account. Don't use Continue with google, instead click on Signup and give an email and password to get get $18 credits for free. After verifying our account on email, we can go on to get our api key.

**Comment by sumit970** [comment]
hey Is there any solution to this ?
please do share

**Comment by Ugyenwangdi** [comment]
> hey Is there any solution to this ? please do share

yes if you just want to test, we get $18 for free when we create a new account

**Comment by jumailj** [comment]
i just added my credit card. now it's working..

**Comment by Saksham1005** [comment]
I just want to ask that even if we want to use our free credits do we have to register our card on the site?

**Comment by Darkcoder-paditya** [comment]
guys, use this:
var openai = new OpenAIApi(configuration);
instead of :
const openai = new OpenAIApi(configuration);

Change const to var


##worked for me

**Comment by Wodaloo** [comment]
anyone find a solution? Changing const to var didnt work for me.

**Comment by Wodaloo** [comment]
just added my credit card to my openai account and now its working

**Comment by akashshukla7458** [comment]
> just added my credit card to my openai account and now its working

have you tried with fresh account, I have 18 credits in my account still getting the error
